### PR TITLE
ts/atr-indicator

### PR DESCRIPTION
**Pull request by sebastianbochan**
Migrated atr indicator to class pattern.

### CODE REVIEW

**Comment by sebastianbochan** [comment]
@TorsteinHonsi @bre1470 
The 3 charts are completely the same, so I we can approve them.

**Review by TorsteinHonsi — COMMENTED** [review]
> The 3 charts are completely the same, so we can approve them.

Yes, this is a problem with the SVG comparison script, we have now fixed it for new forks from the master.

Pending @bre1470 's approval.

File: ts/Stock/Indicators/ATR/ATRIndicator.ts [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+    } as ATROptions);
+
+    /* *
+     *
+     *  Properties
+     *
+     * */
+
+    public data: Array<ATRPoint> = void 0 as any;
+    public points: Array<ATRPoint> = void 0 as any;
```
Comment (bre1470) [inline ts/Stock/Indicators/ATR/ATRIndicator.ts:130]: `options` is missing here.

**Review by bre1470 — CHANGES_REQUESTED** [review]
Thank you! One typed property is missing in the class pattern. See comment.

File: ts/Stock/Indicators/ATR/ATRIndicator.ts (hunk 1 above)
Comment (sebastianbochan) [reply ts/Stock/Indicators/ATR/ATRIndicator.ts:130]: In the interface, I was missed.

Fixed.

**Review by bre1470 — APPROVED** [review]
Thank you! Looks good. 👍

**Pull request merged**